server/src/main/resources/broker.xml

```diff
@@ -97,6 +98,20 @@
                 <redelivery-delay>0</redelivery-delay>
                 <max-delivery-attempts>1</max-delivery-attempts>
             </address-setting>
+
+            <address-setting match="event.jobs">
```

Comment:
Jobs should have its own address and should not be filtered through the 'event'.* path.